client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -93,7 +94,32 @@ export default React.createClass( {
 			} );
 			renewLink = renewLinkPlural;
 		}
-		return <Notice status="is-error" showDismiss={false}>{text} {renewLink}</Notice>;
+		return <Notice status="is-error" showDismiss={ false } key="expiringDomains">{ text } { renewLink }</Notice>;
+	},
+
+	unverifiedDomains() {
+		var notices,
+			domains = this.getDomains().filter( domain => domain.isPendingIcannVerification );
+
+		if ( domains.length > 2 ) {
+			notices = <Notice status="is-error" showDismiss={ false } key="unverifiedDomains">{
+				this.translate( 'Some of your domains have unverified email addresses. This may lead to their temporary suspension.' )
+			}</Notice>;
+		} else if ( domains.length > 0 ) {
+			notices = domains.map( domain => {
+				const text = this.translate( 'You need to verify the email for %(domainName)s or the domain may be suspended. {{a}}Learn more.{{/a}}', {
+					args: {
+						domainName: domain.name
+					},
+					components: {
+						a: <a href={ paths.domainManagementEdit( this.props.selectedSite.domain, domain.name ) } />
```

Comment:
The link says learn more but it goes to the edit page. Maybe we should change it naming or the href itself?